common/common.c

```diff
@@ -1078,6 +1078,38 @@ void destroy_gstring(void *gstring)
 	(void) g_string_free(gstring, TRUE);
 }
 
+#ifdef __MINGW32__
+BT_HIDDEN
+GString *bt_common_normalize_path(const char *path, const char *wd)
+{
+	GString *norm_path;
+	char *tmp;
+
+	assert(path);
+
+	tmp = _fullpath(NULL, path, PATH_MAX);
+	if (!tmp) {
+		goto error;
```

Comment:
Changing this since `norm_path` is uninitialized and will be used at the error label.